@here/harp-mapview/lib/composing/Outline.ts

```diff
@@ -193,16 +246,27 @@ export class OutlineEffect {
     }
 
     private createMaterial(originalMaterial: THREE.Material) {
+        // EdgeMaterial or depth prepass material should not be used for outlines.
+        if (
+            originalMaterial instanceof EdgeMaterial ||
+            (originalMaterial as DepthPrePassProperties) === true
```

Comment:
don't you need to check isDepthPrepassMaterial === true?

Reply:
Big oops... ahem, no: I just tested reviewer attention. Congratulations, you passed!

Sincerely: Thank you for catching this, it is embarrassing.

Reply:
Haha, well it works in the pre-submit anyway, I'll just pretend I didn't see it :)